taskcluster/ac_taskgraph/loader/build_config.py

```diff
@@ -120,7 +120,14 @@ def loader(kind, path, config, params, loaded_tasks):
     # Disable the affected_components optimization to make sure we execute all tests to get
     # a complete code coverage report for pushes to 'main'.
     # See https://github.com/mozilla-mobile/android-components/issues/9382#issuecomment-760506327
-    if params["tasks_for"] in ("github-pull-request", "github-pull-request-untrusted", "github-push") and params["head_ref"] not in ("main", "refs/heads/main"):
+    should_take_out_tasks = all((
+        params["tasks_for"] in ("github-pull-request", "github-pull-request-untrusted", "github-push"),
+        params["head_ref"] not in ("main", "refs/heads/main"),
+        # try_task_config.json can override `target_tasks_method`` to make staging nightly
+        # for instance
+        params["target_tasks_method"] == "default",
```

Comment:
Is this a separate issue / ride-along fix, or is it somehow related to the version number change? I don't object -- just checking my understanding.

Reply:
Thanks for pointing this out! It's a ride-along fix. I wanted to use `try_task_config.json` to make a nightly on this PR but I stumbled upon this bug. That's why I fixed it here.